processors/processPlayerStatus.js

```diff
@@ -2,13 +2,23 @@ const { game_types: gameTypes, modes, maps } = require('hypixelconstants');
 
 const gameTypes_ = new Map(gameTypes
   .map(({ type_name: typeName, clean_name: cleanName }) => [typeName, cleanName]));
-const modes_ = new Map(modes
-  .map(({ key, modes }) => [key, modes])
-  .filter(([_, modes]) => modes)
-  .map(([key, modes]) => [key, new Map(modes.map(({ key, name }) => [key, name]))]));
 const maps_ = new Map(Object.entries(maps)
   .map(([game, maps]) => [game, new Map(Object.entries(maps).map((([map, { name }]) => [map, name])))]));
 
+function getMode(type, mode){
+  let gameObject = modes.find(({key}) => key === type);
+  if (!gameObject) return mode
+  let subGameObject = gameObject.modes.find(
```

Comment:
```suggestion
  const subGameObject = gameObject.modes.find(
```

Maybe these variables can just be called `game` and `subGame`